app/src/androidTest/java/com/github/HumanLearning2021/HumanLearningApp/view/learning/LearningTest.kt

```diff
@@ -25,14 +30,18 @@ import org.junit.Test
 import org.junit.runner.RunWith
 
 // TODO: fix
-@FlakyTest
+//@FlakyTest
```

Comment:
Don't comment out. Remove the annotation and the TODO altogether

Reply:
yep, will do